api/send.py

```diff
@@ -8,6 +8,11 @@
 from msc_apps import *
 import random
 
+#dust_limit=2730
+testnet = False
```

Comment:
@andrewrothman can you pull the latest changes from the staging or master branch into your repo to get the updated syntax that was tweaked.